lib/mj-single.js

```diff
@@ -568,6 +571,7 @@ function GetHTML(result) {
   var script = jax.SourceElement(), html = script.previousSibling;
 
   if (data.speakText) {
+    if (html.querySelector("[aria-label]")){ html.querySelector("[aria-label]").removeAttribute("aria-label");}
```

Comment:
I think it should be `("[aria-label]")) {html` not `("[aria-label]")){ html`, here.  Volker would probably like

```
if (html.querySelector("[aria-label]")) {
  html.querySelector("[aria-label]").removeAttribute("aria-label");
}
```
even better.  Or

```
var label = html.querySelector("[aria-label]");
if (label) label.removeAttribute("aria-label");
```